--- libimobiledevice-1.1.5/cython/lockdown.pxi.orig	2013-03-07 23:05:33.000000000 +0100
+++ libimobiledevice-1.1.5/cython/lockdown.pxi	2013-09-01 10:16:46.775583997 +0200
@@ -219,7 +219,7 @@
         cdef:
             lockdownd_error_t err
             char* c_session_id = NULL
-            bint ssl_enabled
+            int ssl_enabled
             bytes session_id
         err = lockdownd_start_session(self._c_client, host_id, &c_session_id, &ssl_enabled)
         try:
